richtext-ui/src/main/java/com/zachklipp/richtext/ui/SimpleTableLayout.kt

```diff
@@ -61,7 +61,7 @@ internal fun SimpleTableLayout(
     //   // Divide the height by the number of rows, then leave room for the padding.
     //   (constraints.maxHeight - cellSpacingHeight) / rowMeasurables.size
     // }
-    val cellConstraints = constraints.constrain(Constraints(maxWidth = cellWidth.roundToInt()))
+    val cellConstraints = Constraints(maxWidth = cellWidth.roundToInt()).constrain(constraints)
```

Comment:
The way this function worked actually surprised me as well. There is something not right either with the wording or the order. 

```
constraints.constrainedBy(otherConstraints)
```

This would have been better IMO 🤷

Reply:
Yea, I think that would be much better. Wanna file it?